tensorflow/stream_executor/cuda/cuda_driver.cc

```diff
@@ -1216,7 +1189,34 @@ GpuDriver::CreateMemoryHandle(GpuContext* context, uint64 bytes) {
                                                    uint64 size,
                                                    CUstream stream) {
   ScopedActivateContext activation(context);
-  CUresult result = cuMemcpyDtoDAsync(gpu_dst, gpu_src, size, stream);
+  CUresult result;
+  // CreatedContexts::GetAnyContext() doesn't works when ptr == 0.
```

Comment:
Would it make more sense to skip the copy altogether when size is 0?

Reply:
I asked myself this question. But the old code path didn't had that and I wasn't totally sure if it could mess with the dependency on the compute vs transfer stream. So I went for the safe thing to not do that.